extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -644,39 +662,38 @@ export class AmpPanZoom extends AMP.BaseElement {
   }
 
   /**
-   * Performs a one-step pinch zoom action.
-   * @param {number} centerClientX
-   * @param {number} centerClientY
-   * @param {number} deltaX
-   * @param {number} deltaY
-   *  @param {number} dir
-   * @private
+   * @param {number} clientX
+   * @param {number} clientY
    */
-  onPinchZoom_(centerClientX, centerClientY, deltaX, deltaY, dir) {
-    this.zoomToPoint_(centerClientX, centerClientY, deltaX, deltaY, dir);
+  onDoubletapZoom_(clientX, clientY) {
+    const newScale = this.scale_ == this.minScale_ ?
+      this.maxScale_ : this.minScale_;
+    const dx = (this.elementBox_.width / 2) + this.getOffsetX_(clientX);
+    const dy = (this.elementBox_.height / 2) + this.getOffsetY_(clientY);
+    return this.onZoom_(newScale, dx, dy, /*animate*/ true);
   }
 
   /**
-   * Given center position, zoom delta, and zoom position, computes
-   * and updates a zoom action on the content.
+   * Performs a one-step pinch zoom action.
    * @param {number} centerClientX
    * @param {number} centerClientY
    * @param {number} deltaX
    * @param {number} deltaY
-   * @param {number} dir
+   *  @param {number} dir
```

Comment:
An extra space is inserted here, should remove it

Reply:
Done